app/src/redux-resources/analytics/hooks/useCameraAnalytics.ts

```diff
@@ -0,0 +1,134 @@
+import {
+  ANALYTICS_CAMERA_ENABLEMENT_KIND,
+  ANALYTICS_CAMERA_SETTINGS_KIND,
+  ANALYTICS_IMAGE_CAPTURE_KIND,
+  ANALYTICS_LIVE_FEED_KIND,
+  ANALYTICS_PHOTO_ACCESS,
+  useTrackEvent,
+} from '/app/redux/analytics'
+
+import type { RobotType } from '@opentrons/shared-data'
+
+export interface cameraAnalyticsParams {
```

Comment:
```suggestion
export interface CameraAnalyticsParams {
```
All these types should be capitalized